package.json

```diff
@@ -36,13 +36,39 @@
     ".": {
       "import": {
         "types": "./esm/index.d.mts",
+        "react-server": "./esm/vanilla.mjs",
         "default": "./esm/index.mjs"
       },
       "default": {
         "types": "./index.d.ts",
+        "react-server": "./vanilla.js",
         "default": "./index.js"
       }
     },
+    "./react": {
+      "import": {
+        "types": "./esm/react.d.mts",
+        "react-server": "./esm/error.mjs",
+        "default": "./esm/react.mjs"
+      },
+      "default": {
+        "react-server": "./error.js",
+        "types": "./react.d.ts",
+        "default": "./react.js"
+      }
+    },
+    "./react/*": {
+      "import": {
+        "types": "./esm/*.d.mts",
+        "react-server": "./esm/error.mjs",
+        "default": "./esm/*.mjs"
+      },
+      "default": {
+        "types": "./*.d.ts",
+        "react-server": "./error.js",
+        "default": "./*.js"
+      }
+    },
```

Comment:
I go through the code, and all react* files uses `useContext` or `useAtom` which is not accepted in react server environment